microcosm_sqlite/stores.py

```diff
@@ -16,11 +16,49 @@
 )
 
 
+def get_session(store):
```

Comment:
This is our default logic for finding a session to use in a store. It assume the session is managed externally, e.g. via:

```
with SessionContext(graph, data_set):
     store.foo()
```